notifications.yaml

```diff
@@ -0,0 +1,220 @@
+swagger: "2.0"
+info: 
+  version: "0.0.1"
+  title: "Notification API"
+  description: "Notification API."
+host: "localhost"
+basePath: "/api/v1"
+schemes: 
+  - "https"
+paths: 
+  /Messages/{id}: 
+    parameters: 
+      - 
+        name: "id"
```

Comment:
According to Zalando's guidelines "id" *must* be the standard name for an identifier:

https://zalando.github.io/restful-api-guidelines/common-data-objects/CommonDataObjects.html